test/definitions/divorce/CaseTypeTab.test.js

```diff
@@ -0,0 +1,112 @@
+const expect = require('chai').expect;
+const assert = require('chai').assert;
+const _ = require('loadsh');
+const caseTypeTab = require('../../../definitions/divorce/json/CaseTypeTab');
+const caseField = require('../../../definitions/divorce/json/CaseField');
+describe('CaseTypeTab', function() {
+  describe('duplicates', function() {
+    it('should contain a unique case field ID per tab ID (no duplicate field in a tab)', function() {
+      const uniqResult = _.uniqWith(
+        caseTypeTab,
+        (field1, field2) =>
+          field1.TabID === field2.TabID &&
+          field1.CaseFieldID === field2.CaseFieldID
+      );
+      expect(uniqResult).to.eql(caseTypeTab);
+    });
+    it('should contain a unique tab field display order ID field tab ID (no duplicate field order in a tab)', function() {
+      const tabIds = _.uniq(_.map(caseTypeTab, 'TabID'));
+      tabIds.forEach(tabId => {
+        const allFieldsPerTab = _.filter(caseTypeTab, (field) => {
+          return field.TabID === tabId;
+        });
```

Comment:
```
const allFieldsPerTab = _.filter(caseTypeTab, field => field.TabID === tabId);
```

Reply:
I tried this, but with `eslint --fix` it keeps reverting it to the longer format, any idea?

Reply:
We can use an es-lint override to allow this but it's up to you